Misc/NEWS.d/next/Library/2022-07-03-22-58-42.gh-issue-94512.W-g4ff.rst

```diff
@@ -0,0 +1 @@
+Convert :mod:`winreg` to use Argument Clinic.
```

Comment:
It was already using it?

If this is a completely invisible change to users, we don't need a NEWS entry.

Reply:
Thank you, done.